### ISSUE TITLE
Test failures (TestNode, TestNodeID)

### ISSUE THREAD

**Issue by onlyjob** [google/uuid#31]
I'm getting the following test failures with Golang 1.10:

~~~~
=== RUN   TestNode
--- FAIL: TestNode (0.00s)
        uuid_test.go:306: NodeInterface returned an empty string
=== RUN   TestNodeID
--- FAIL: TestNodeID (0.00s)
        uuid_test.go:376: NodeInterface "user" after SetInteface
~~~~

**Comment by fedir** [comment]
All works for me.

```
$ go version
go version go1.10.3 darwin/amd64
$ go test
PASS
ok  	github.com/google/uuid	0.011s
```

**Comment by darkfeline** [comment]
These tests fail if the running environment doesn't have any network interfaces (e.g., container, VM).

**Comment by darkfeline** [comment]
On Linux, you can simulate this with:

```
sudo ip netns add tmp
sudo ip netns exec tmp go test ./...
```

**Comment by pborman** [comment]
I expect darkfeline's change to be incorporated into v1.1 which might not be that far out.

**Comment by pborman** [comment]
Closing this as V1.1 has the changes.

**Comment by carlwgeorge** [comment]
@pborman #34 has not been merged yet, and [v1.1.0 does not have the change](https://github.com/google/uuid/blob/v1.1.0/node.go#L50-L53).  Did you mean a different tag?

**Comment by pborman** [comment]
Hmm, let me take a look.

**Comment by pborman** [comment]
Okay, I just published v1.1.1 _with_ darkfeline's fixes.  Tested on Linux with suggest code snippets above.
